# @author Maik Basso <maik@maikbasso.com.br>

# message header 
int32 id
int32 source_node_id
int32 destination_node_id
int32 message_id
